.goreleaser.yml

```diff
@@ -11,6 +11,8 @@ builds:
       - amd64
       - 386
       - arm64
+    env:
+      - CGO_ENABLED=0
```

Comment:
I had to add this because binaries are now being built on host OS with Go cross-complier and then copy to the Docker image. Therefore, the applications that have been built on e.g. Ubuntu don't run inside Alpine image.
`CGO_ENABLED=0` enables static linking for binaries and makes them more portable.